src/components/pages/footer.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import './Footer.css'
+
+function Footer () {
```

Comment:
Can we keep the name of the component file in capitals?

viz. footer.jsx → Footer.jsx